### PR TITLE
Create Introduction-to-Cloud-Native.md

### PR DESCRIPTION
**Pull request by AmitKarnam**
Added introduction to cloud-native article.

### CODE REVIEW

File: resources/Introduction-to-Cloud-Native.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Introduction to Cloud Native #
+
+### **What is Cloud Native?**  <hr/>
+
+***The Cloud Native Computing Foundation(CNCF)***  defines Cloud Native as technologies that empower organizations to build and run scalable applications in modern, dynamic environments 
+such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
+
+**So simply defined, cloud-native is building an application for the cloud**
```
Comment (AbhijithGanesh) [inline resources/Introduction-to-Cloud-Native.md:8]: I think for the cloud is technically infrastructure related, maybe if it is rephrased it'd be better.

File: resources/Introduction-to-Cloud-Native.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+The applications that are designed traditionally  are not built keeping cloud compatibility in mind, this hinders the smooth functioning of the application.<br/>
+
+2) So, we need a new architecture for our applications to utilize the benefits of the cloud. There is a need to design the applications keeping the cloud in mind and take advantage of several services the cloud provides.<br/>
+
+
+
+### **Why Cloud Native?**   <hr/>
+
+Since the cloud reduces the overhead cost of owning a server and configuring it to run our application, it makes more sense to use a cloud service to run our application which makes the whole process easy.
+The cloud offers various services that can be utilied to make our applications more efficient.<br/><br/>
```
Comment (AbhijithGanesh) [inline resources/Introduction-to-Cloud-Native.md:22]: It'd also be great if we can emphasize on the fact that applications built on the philosophy and principles of cloud native can still be extended to the `on-premises` server. Cloud Native gives us that extended flexibility. An application that is deployed on cloud can be deployed on-prem if it is managed well and done right.

**Comment by AbhijithGanesh** [comment]
The comments are suggestions, feel free to use them. Apart from those, it looks good to me.

File: resources/Introduction-to-Cloud-Native.md (hunk 2 above)
Comment (AmitKarnam) [reply resources/Introduction-to-Cloud-Native.md:22]: This is a good point, I'll add that.

File: resources/Introduction-to-Cloud-Native.md (hunk 1 above)
Comment (AmitKarnam) [reply resources/Introduction-to-Cloud-Native.md:8]: Can you elaborate this point? The last statement I guess must be rephrased , but the above is a definition taken from CNCF glossary. 
Link : [ CNCF Glossary Link](https://github.com/cncf/toc/blob/main/DEFINITION.md )

**Review by shreemaan-abhishek — APPROVED** [review]
LGTM

**Review by shreemaan-abhishek — APPROVED** [review]

**Review by yashgangwar7558 — APPROVED** [review]

**Review by AbhijithGanesh — APPROVED** [review]

**Review by helper-uttam — APPROVED** [review]

**Comment by AbhijithGanesh** [comment]
@patchandpray Can you merge this?

**Review by patchandpray — APPROVED** [review]

**Pull request merged**